index.js

```diff
@@ -29,15 +29,20 @@ async function getConfig() {
     symbol: false,
     skipQuestions: [''],
     subjectMaxLength: 75,
-    conventional: false
+    format: '{emoji} {scope} {subject}',
```

Comment:
Removing `conventional` is a breaking change I think we could avoid. I would put it back and handle it in `format` (cf. comment below).